cookbooks/fb_logrotate/README.md

```diff
@@ -224,15 +224,20 @@ of CentOS6 and MAC OS X. It will remove the cronjob for logrotate
 adds low priority `nice` and `ionice` to the `logrotate` process. This can be
 very useful, but be aware you are removing a file from the system package.
 
-### timer_settings
-The node['fb_logrotate']['timer_settings'] contains configuration for the
+### systemd_setting
+The node['fb_logrotate']['systemd_settings'] contains configuration for the
 systemd timer and service units. It accepts the following:
 * OnCalendar - when and how often the logrotate should run. Default 'daily'
 * RandomizedDelaySec - Delay the timer by a randomly selected, evenly
                        distributed amount of time between 0 and the specified
                        time value. Default 0.
 * Nice - Nice level for the logrotate process. Default 19 (lowest priority).
 * IOSchedulingClass - I/O scheduling class for executed processes. Default 3 (idle).
+* Persistent - Boolean.  If true, the last time the timer ran is stored on disk.
+               When the timer is activated, the service unit is triggered immediately
+               if it would have been triggered at least once during the time when
+               the timer was inactive.  Useful for making up runs
+               on machines that have been powered down.  Default false.
```

Comment:
Please don't double-space after punctuation. Ignoring the argument of whether or not that is relevant anymore, it's inconsistent with the rest of this codebase.

Reply:
@jaymzh Should be fixed in https://github.com/facebook/chef-cookbooks/pull/326/commits/d62e5ce785a02a56f430b914e86e70405ffed7b1!